parlai/core/utils.py

```diff
@@ -33,6 +33,14 @@
     __TORCH_AVAILABLE = False
 
 
+def check_torch_version():
```

Comment:
i think we need to get rid of the try/except above as well, and perhaps move into this function as well?

Reply:
Made some changes in the new commit.